data/test.go

```diff
@@ -0,0 +1,15 @@
+package data
+
+import "gopkg.in/toqueteos/substring.v1"
+
+var TestMatchers = substring.Or(
```

Comment:
So far, all things under the `data` package were auto-generated from Linguist.

As this change exposes a new global var in this package, it looks like a good opportunity to document it, at least mentioning the fact that this does not come from Linguist (even if there is no generated header, that still looks useful for the future maintenance).